server/src/forwarders/socketForwarder.ts

```diff
@@ -241,6 +247,17 @@ class SocketForwarder {
         const ent: HaEntityStateChanged = eventData;
         if (ent.entity_id.startsWith('automation')) {
             this.notifyObservers('automationUpdated', ent.entity_id);
+        } else if (ent.entity_id.startsWith('group')) {
+            let event: Event;
+            if (ent.old_state !== null && ent.new_state !== null) {
+                event = 'groupUpdated';
+            } else if (!ent.old_state === null && ent.new_state !== null) {
```

Comment:
Is it me or those two conditions are the same ?

Reply:
yes but read an outdated file